amundsen_application/static/js/ducks/search/sagas.ts

```diff
@@ -42,6 +42,21 @@ import {
 import { autoSelectResource, getPageIndex, getSearchState } from './utils';
 import { updateSearchUrl } from 'utils/navigation-utils';
 
+import {
+  UpdateSearchFilter
+} from './filters/reducer';
+
+export function* filterWorker(action: any): SagaIterator {
+  const state = yield select();
+  yield put(searchResource(state.search.search_term, state.search.selectedTab, getPageIndex(state.search)));
+};
+export function* multiSelectFilterWatcher(): SagaIterator {
+  yield debounce(750, [UpdateSearchFilter.ADD_MULTI_SELECT, UpdateSearchFilter.REMOVE_MULTI_SELECT], filterWorker);
```

Comment:
Does the `debounce` also guarantee we only take the latest response? It's probably not an issue with our current low latency, however.

Reply:
Yes, unless I'm misinterpreting the doc. https://github.com/redux-saga/redux-saga/blob/master/docs/recipes/README.md#debouncing